app/content/views/strike.py

```diff
@@ -16,9 +17,14 @@
 class StrikeViewSet(viewsets.ModelViewSet):
     serializer_class = StrikeSerializer
     queryset = Strike.objects.all()
+    permission_classes = [BasicViewPermission]
 
     def get_queryset(self):
-        return (strike for strike in Strike.objects.all() if strike.active)
+        return (
+            (strike for strike in Strike.objects.all() if strike.active)
```

Comment:
Denne spørringen vil bli ganske spenstig etter hvert, men er greit for nå

Reply:
tenker vi bytter den ut når din PR kommer inn